lib/ecto.ex

```diff
@@ -355,7 +355,7 @@ defmodule Ecto do
 
   ### Embeds
 
-  Ecto also supports embeds. While associations keep parent and child
+  Ecto also supports [embeds](https://docs.mongodb.com/manual/core/data-model-design/#data-modeling-embedding). While associations keep parent and child
```

Comment:
Yeah, I had my concerns about that too but I couldn't find any DB-agnostic description of it such as a wikipedia entry. Thanks again and happy new year!